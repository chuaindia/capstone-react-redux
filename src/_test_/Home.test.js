import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Coin from '../components/Home';
import '@testing-library/jest-dom';
import store from '../redux/configureStore';

it('renders correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Coin />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});