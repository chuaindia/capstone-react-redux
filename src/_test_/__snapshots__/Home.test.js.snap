// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="TableContainer"
>
  <div>
    <h1
      className="Header"
    >
      Crypto Currencies
    </h1>
  </div>
  <table>
    <div
      className="CoinContainer"
    />
  </table>
</div>
`;
